### ISSUE TITLE
Data duplication in the dataset

### ISSUE THREAD

**Issue by Aavash232311** [Aavash232311/transformer-pt-analysis#1]
Data duplication in the dataset

When evaluating, we are seeing duplicated data. When I was trying to plot a heat map of the raw logits before continuing to diagonal spectral mass, it became noisy. Even though that might be in the evaluation phase, to obtain an nxn lattice, we need to perform a forward pass, and we might not see the signal that we expect from our analysis. 

mod = 10

sequence: 1,1,2,3,5,8,3,... repeats
